microprofile-reactive-messaging-kafka/xp-install-rhosak.adoc

```diff
@@ -11,7 +11,7 @@ We will summarize the steps required to set up a Kafka instance in RHOSAK here.
 
```

Comment:
@kabir just a note to the commit message - not sure whether `The operator to install RHOSAK  is now called RHOAS` is completely correct. Per the description of the `RHOAS` operator, it's Red Hat OpenShift Application Services Operator - so it's probably supposed to work as a utility for more than just a RHOSAK product. RHOSAK is probably only connected product at this point of development... but since this commit will be probably squashed into the one causing this commit message to disappear anyway, I don't know why I'm writing this here... :D